lib/Collection.js

```diff
@@ -97,6 +97,25 @@ module.exports = class Collection {
     if(!(id || type || equals || has)) {
       throw new TypeError('"id", "type", "equals", or "has" must be given.');
     }
+    equals = this._prepareEqualsQuery({id, type, equals});
+    const {capability, edvClient, invocationSigner} = this;
+    const results = await edvClient.find(
+      {equals, has, capability, invocationSigner});
+    return results;
+  }
+
+  async countDocuments({id, type, equals, has}) {
+    if(!(id || type || equals || has)) {
+      throw new TypeError('"id", "type", "equals", or "has" must be given.');
+    }
+    equals = this._prepareEqualsQuery({id, type, equals});
+    const {capability, edvClient, invocationSigner} = this;
+    const documentCount = await edvClient.count(
+      {equals, has, capability, invocationSigner});
+    return documentCount;
+  }
+
+  _prepareEqualsQuery({id, type, equals}) {
```

Comment:
```suggestion
  _prepareEqualsQuery({id, type, equals = []}) {
```
With this the conditional can be simplified removing the `!equals` case because it will be defaulted via destructuring assignment.